Update Puppet documents (#2847)
wazuh/wazuh-documentation: source/deploying-with-puppet/wazuh-puppet-module/reference-wazuh-puppet/wazuh-agent-class.rst

  `Default 500`

$ossec_config_profiles
  Specify the agent.conf profile(s) to be used by the agent.

.. _ref_agent_vars_localfile:

Localfile variables
-------------------

$ossec_local_files
  Files list for log analysis

  This files are listed in params_agent.pp in section $default_local_files

.. _ref_agent_vars_rootcheck:

Rootcheck variables
-------------------

$configure_rootcheck
  Enables rootcheck section render on this host.

  `Default true`

$ossec_rootcheck_disabled
  Disable rootcheck on this host (Linux).

  `Default no`

$ossec_rootcheck_check_files
  Enable rootcheck checkfiles option.

  `Default yes`

$ossec_rootcheck_check_trojans
  Enable rootcheck checktrojans option.

  `Default yes`

$ossec_rootcheck_check_dev
  Enable rootcheck checkdev option.

  `Default yes`

$ossec_rootcheck_check_sys
  Enable rootcheck checksys option.

  `Default yes`

$ossec_rootcheck_check_pids
  Enable rootcheck checkpids option.

  `Default yes`

$ossec_rootcheck_check_ports
  Enable rootcheck checkports option.

  `Default yes`

$ossec_rootcheck_check_if
  Enable rootcheck checkif option.

  `Default yes`

$ossec_rootcheck_frequency

>   Frequency that the rootcheck is going to be executed (in seconds).

Reviewer: ```suggestion
  How often the rootcheck scan will run (in seconds).
```

Author: Done on 2faa2ea, thanks !

  `Default true`

$configure_sca
  Enables sca section render on this host.

  `Default true`

$sca_amazon_enabled
  Enable SCA on this host (Amazon Linux 2).

  `Default yes`

$sca_amazon_scan_on_start
  The SCA module will perform the scan immediately when started (Amazon Linux 2).

  `Default yes`

$sca_amazon_interval
  Interval between module executions.

  `Default 12h`

$sca_amazon_skip_nfs
  Enable or disable the scanning of network mounted filesystems (Works on Linux and FreeBSD). Currently, `skip_nfs` will exclude checking files on CIFS or NFS mounts.

  `Default yes`

$sca_amazon_policies
  In this section it can be included a list of policy files to run assessments.

  `Default []`

$sca_rhel_enabled
  Enable SCA on this host (RHEL).

  `Default yes`

$sca_rhel_scan_on_start
  The SCA module will perform the scan immediately when started (RHEL).

  `Default yes`

$sca_rhel_interval
  Interval between module executions.

  `Default 12h`

$sca_rhel_skip_nfs
  Enable or disable the scanning of network mounted filesystems (Works on Linux and FreeBSD). Currently, `skip_nfs` will exclude checking files on CIFS or NFS mounts.

  `Default yes`

$sca_rhel_policies

>   In this section it can be included a list of policy files to run assessments.

Reviewer: ```suggestion
  A list of policies to run assessments can be included in this section.
```

Author: Done on 2faa2ea, thanks !
